prov/efa/src/rdm/efa_rdm_ep_fiops.c

```diff
@@ -466,6 +466,12 @@ void efa_rdm_ep_set_use_zcpy_rx(struct efa_rdm_ep *ep)
 		goto out;
 	}
 
+	if (ep->shm_ep) {
+		EFA_INFO(FI_LOG_EP_CTRL, "Libfabric SHM is not turned off, zero-copy receive protocol will be disabled\n");
```

Comment:
Shall we also document this limitation on man page?

Reply:
Yes, updated the rdm protocol doc